pmoves/services/creator-operator/tests/test_operator.py

```
@@ -0,0 +1,30 @@
+import pytest
+from operator_helpers import parse_workorder, assemble_result
+from fixtures import VALID_WORKORDER
+
+
+def test_parse_workorder_validates():
+    wo = parse_workorder(VALID_WORKORDER)
+    assert wo["workflow_id"] == "image.ideogram-ultra"
+
+
+def test_parse_workorder_rejects_bad():
+    with pytest.raises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)